### PR TITLE
Create my profile screen

### PR DESCRIPTION
**Pull request by zoha565**
## Description:
1) Gave autolayout to the entire screen
2) Created an extension for the view controller and did all the UI customizations in it.

## Screenshots/gifts: 

Simulator Screenshot - iPhone 14 Pro - 2023-08-04 at 15 54 03

### CODE REVIEW

**Review by zuhair5 — CHANGES_REQUESTED** [review]
- Your createprofile+extension file is not shown kindly recheck it and commit that too
- The autolayout is not correct it should not be showing any errors which i am seeing 
- The profile picture has problems with it's corner radius, either the picture is too big or something else, plus try using a picture in **Png** format. 
- Lastly you don't need the tab bar controller at the bottom as it is a separate story so kindly remove that, it is my mistake i should have mentioned it in the story. 

**tab bar screenshot**
Screenshot 2023-08-04 at 4 33 09 PM


**Kindly fix all the issues mentioned above and re-commit.**

**Comment by zoha565** [comment]
fixed

File: ShopGo/CreateMyProfile/CreateProfile+extension.swift [hunk 1]
```diff
@@ -13,14 +13,11 @@ extension CreateMyProfileViewController{
...
         setting.image = UIImage(named: "setting")
-        home.image = UIImage(named: "home")
-        cart.image = UIImage(named: "cartIcon")
-        profile.image = UIImage(named: "profile")
         profileImage.image = UIImage(named: "ProfilePic")
     }
     
     func profilePictureUI(){
-        profileImage.layer.cornerRadius = 20
+        profileImage.layer.cornerRadius = 15
```
Comment (aliahmed1105) [inline ShopGo/CreateMyProfile/CreateProfile+extension.swift]: Remove this extra space.

File: ShopGo/CreateMyProfile/CreateProfile+extension.swift
Comment (aliahmed1105) [inline ShopGo/CreateMyProfile/CreateProfile+extension.swift:1]: I would suggest adding space after import UIKit.

**Review by haroonmaqsoodch — COMMENTED** [review]
Screenshot 2023-08-05 at 2 08 09 PM

These should be buttons not labels. We are going to use them for navigation to other screens. Change them to buttons and create their connections as well.

**Comment by zoha565** [comment]
fixed

File: ShopGo/CreateMyProfile/CreateProfile+extension.swift
Comment (zoha565) [reply ShopGo/CreateMyProfile/CreateProfile+extension.swift:1]: fixed

File: ShopGo/CreateMyProfile/CreateProfile+extension.swift (hunk 1 above)
Comment (zoha565) [reply ShopGo/CreateMyProfile/CreateProfile+extension.swift]: fixed

**Review by zuhair5 — APPROVED** [review]
**LGTM**

**Review by haroonmaqsoodch — APPROVED** [review]

**Pull request merged**